cmd/connectconformance/main.go

```diff
@@ -219,8 +219,6 @@ func run(flags *flags, cobraFlags *pflag.FlagSet, command []string) { //nolint:g
 			fatal(`Server command (after the "----") is empty.`)
 		}
 	default:
-		// TODO: support mode "both", which would allow the caller to supply both the
```

Comment:
Oops, the TODO was already addressed, but just forgot to remove it in earlier change.